src/unix/fs.c

```diff
@@ -1180,7 +1181,10 @@ static ssize_t uv__fs_copyfile(uv_fs_t* req) {
   bytes_to_send = src_statsbuf.st_size;
   in_offset = 0;
   while (bytes_to_send != 0) {
-    uv_fs_sendfile(NULL, &fs_req, dstfd, srcfd, in_offset, bytes_to_send, NULL);
+    bytes_chunk = SSIZE_MAX;
+    if (bytes_to_send < bytes_chunk)
```

Comment:
This is generating a `-Wsign-compare` warning because `off_t` is signed. I _think_ the right way to fix that is:
```suggestion
    if (bytes_to_send < (off_t) bytes_chunk)
```
Because it's conceivable that `sizeof(size_t) < sizeof(off_t)` but only in an alternate universe would the reverse be true. Agree/disagree?

Reply:
I agree

Reply:
I added another commit that fixes this